.claude/scripts/keyserve.py

```
@@ -0,0 +1,88 @@
+
+        def do_OPTIONS(self) -> None:  # noqa: N802 - BaseHTTPRequestHandler contract.
+            self.send_response(204); self._cors(); self.end_headers()
+
+        def do_GET(self) -> None:  # noqa: N802 - BaseHTTPRequestHandler contract.
+            if self.path != '/' + token:
+                self.send_response(404); self._cors(); self.end_headers(); return
+            body = json.dumps({'value': value}).encode()
+            self.send_response(200)
+            self.send_header('Content-Type', 'application/json')
+            self.send_header('Content-Length', str(len(body)))
+            self._cors(); self.end_headers()
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)